src/js/components/Select/index.d.ts

```diff
@@ -34,7 +34,7 @@ export interface BasicSelectProps {
   name?: string;
   onChange?: (...args: any[]) => void;
   onClose?: (...args: any[]) => any;
-  onMore?: (...args: any[]) => any;
+  onMore?: () => void | boolean;
```

Comment:
I would expect onMore to be a function

Reply:
I originally thought that onMore could be a boolean because it's used within an useEffect hook in the src\js\components\InfiniteScroll\InfiniteScroll.js file. However, I have realized that onMore is actually declared as a function. This means that onMore must be a function in order for the useEffect hook to execute the code block within the if condition.

Thanks.